flax/core/frozen_dict.py

```diff
@@ -24,6 +24,12 @@
 V = TypeVar('V')
 
 
+def _indent(x, num_spaces):
+  indent_str = ' ' * num_spaces
+  lines = x.split('\n')
+  return '\n'.join(indent_str + line for line in lines[:-1]) + '\n'
```

Comment:
Maybe useful to add a comment here explaining `lines[:-1]` strips off the trailing `\n`.

Reply:
Done